app/client/cypress/integration/national/national.spec.js

```diff
@@ -0,0 +1,189 @@
+describe('State Links', () => {
+  beforeEach(() => {
+    cy.visit('/national');
+  });
+
+  it('Clicking the “DISCLAIMER” button displays the disclaimer popup', () => {
+    // verify opening the disclaimer
+    cy.findByText('Disclaimer').click();
+    cy.findByText('EPA has posted information through this', {
+      exact: false,
+    }).should('exist');
+
+    // verify closing the disclaimer
+    cy.findByTitle('Close disclaimer').click();
+    cy.findByText('EPA has posted information through this', {
+      exact: false,
+    }).should('not.exist');
+  });
+
+  it('Clicking the "National Drinking Water" tab changes the tab and does not update the Url', () => {
+    // verify tab changed correctly
+    cy.findByTestId('hmw-national-drinking-water-tab').click();
+
+    // verify National Drinking Water tab title is displayed
+    cy.findByTestId('hmw-national-drinking-water-tab')
+      .findByText('Explore National Drinking Water Information')
+      .should('be.visible');
+
+    // verify Url stayed the same
+    cy.url().should('equal', `${document.location.origin}/national`);
+  });
+
+  it('Switching the sub-tabs updates the content within the container', () => {
+    // verify switching to coasts tab changes the content
+    cy.findByTestId('hmw-national-drinking-water-tab').click();
+
+    cy.findByTestId('hmw-national-drinking-water-tab')
+      .findByText('of our coasts are healthy based on biological communities')
+      .should('exist');
+
+    // verify first tab contents are hidden
+    cy.findByTestId('hmw-national-water-conditions-tab')
+      .findByText(
+        'of our rivers and streams are healthy based on biological communities',
+      )
+      .should('not.be.visible');
+  });
+
+  it('From one of the sub-tabs, clicking “Expand All/Collapse All” expands/collapses the content', () => {
+    // verify that only "Expand All" is visible
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('not.exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('exist');
```

Comment:
@cschwinderg for the other requested changes I adjusted the pattern of getting elements and filtering them to instead use data-testids.  

But for these Expand/Collapse All buttons I ran into an issue, There are 4 of these buttons on the National page because they come from the Accordion component. I'm using FindAllByTestId and then filtering by visible items, which ends up being the same number of lines. I think this is still an improvement but I'm open to any ideas you have

Reply:
This looks good to me, since it is no longer getting all of the buttons on the screen. 

Thank you for making the changes.